### PR TITLE
Reduce usage of Imports

### PR DESCRIPTION
**Pull request by shanmukhaditya**
Completed #114

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -52,7 +52,6 @@ provider:
 plugins:
   - serverless-python-requirements
   - serverless-offline
-  - serverless-dotenv-plugin
```
Comment (js324) [inline serverless.yml:55]: Add this back in

File: Dockerfile [hunk 2]
```diff
@@ -20,7 +20,7 @@ ADD package.json /
 
 # Install the node packages outlined in package-lock.json (if present) or package.json
 RUN npm install
-RUN npm install -g serverless
+RUN npm install -g serverless@2.72.2
```
Comment (js324) [inline Dockerfile:23]: Just to keep it clean, remove the versioning "@2.72.2" from this line

**Review by js324 — CHANGES_REQUESTED** [review]
Looks good! Just some minor fixes to docker file and serverless.yml

File: Dockerfile (hunk 2 above)
Comment (shanmukhaditya) [reply Dockerfile]: Hey Jin. I have added those lines back. Can you check now?

**Review by js324 — APPROVED** [review]

**Pull request merged**